scipy/stats/_generate_pyx.py

```diff
@@ -0,0 +1,15 @@
+import pathlib
+import numpy as np
+
+def make_biasedurn():
+    # numpy's random C API changes between 17 and 18
+    NPY_OLD = int(np.__version__.split('.')[1]) < 17
```

Comment:
@chrisb83 Do you know if the break is numpy <1.17 or <1.18?  I think I might have gotten this wrong (even if the CI might pass...)

Reply:
From what I know, the new random generators were introduced in 1.17 (https://numpy.org/doc/stable/release/1.17.0-notes.html) but the cimport only works in 1.18 and later